packages/workspace/src/generators/move/lib/update-imports.ts

```diff
@@ -148,8 +149,8 @@ export function updateImports(
       to: schema.relativeToRootDestination,
     };
 
-    if (tsConfig) {
-      const path = tsConfig.compilerOptions.paths[projectRef.from] as string[];
+    if (tsConfig && paths.length > 0) {
```

Comment:
I think moving `writeJson(...)` into the `if (tsConfig)` is enough to fix the issue. 

Updating `getRootTsConfigPathInTree` is fine since it's only used here and in the remove generator. But we should also change the check to `tree.exists(tsConfigPath)` to account for `null`.

Reply:
That thought did cross my mind, but I was worried about false positives. I'll remove the second clause all the same, though. I'll also go ahead and take a look at `tree.exists(tsConfigPath)` so that it'll account for `null`.

I also believe I can greatly clean up the code quite a bit, too, so that it scales far better. I just wanted to hold off on it in favor of getting these essential changes merged quickly first before with as few changes as possible.

Reply:
I looked into updating `tree.exists(filePath)` to change its argument typing to `string | null` directly without just adding on another if-operator clause.

From what I can tell, it would require touching 3 additional files _(with starting line numbers)_:
1. `packages/devkit/src/utils/invoke-nx-generator.ts:L131`
2. `packages/nx/src/generators/tree.ts:L62`
3. `e2e/utils/file-utils.ts:L111`

The changes seem fairly straight forward, but since this effects a critical Nx component, I figured we could save it for a second PR that I can submit right after this one gets approved. If anything else, just to keep the PRs better focused.

With that said, I applied the necessary changes locally on my machine, and all of the tests appear to be passing on my end.

Reply:
Hey @jaysoo! I merged your changes in with mine, so it should be good to go now.